archetypes/business-service-jaxrs/pom.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <groupId>org.apache.servicecomb.archetypes</groupId>
+  <artifactId>business-service-jaxrs</artifactId>
+  <modelVersion>4.0.0</modelVersion>
+  <version>1.0.0-m2-SNAPSHOT</version>
+
+  <properties>
+    <java-chassis.version>1.0.0-m2-SNAPSHOT</java-chassis.version>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.apache.servicecomb</groupId>
+        <artifactId>java-chassis-dependencies</artifactId>
+        <version>${java-chassis.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <!--for provider-->
+    <dependency>
+      <groupId>org.apache.servicecomb</groupId>
```

Comment:
We can simplify user project dependency by providing some mechanism like spring boot starter. In CSE solution, we called solutions, in witch we created POM dependencies , handler chain defaults and so on. You can find samples of cse-solution-service-engine here : https://github.com/huawei-microservice-demo/porter/blob/master/file-service/pom.xml

Reply:
it's better to use the ServiceComb  dependencies for all the audience.

Reply:
Yes. I just show an example. Maybe we can create one pom like solution-servicecomb-rest and users code need only one dependency.